extensions/amazon-aws/src/components/searchbar/CloudwatchLogsTimeDropdown.tsx

```diff
@@ -0,0 +1,26 @@
+import { LogStartTimes } from "../../interfaces";
+import { List } from "@raycast/api";
+import { useEffect, useState } from "react";
+
+function CloudwatchLogsTimeDropdown({ onChange }: { onChange(logStartTime: LogStartTimes): void }) {
+  const [logStartTime, setLogStartTime] = useState<LogStartTimes>(LogStartTimes.OneHour);
+
+  useEffect(() => {
+    onChange(logStartTime);
+  }, [logStartTime]);
```

Comment:
I suggest making this component fully controlled, since the parent is already keeping the `logStartTime` value.
```suggestion
function CloudwatchLogsTimeDropdown({ value, onChange }: { value: LogStartTimes; onChange(logStartTime: LogStartTimes): void }) {
```

Reply:
Good point, it went through several iterations / cleanups to reach that point so it evaded me.